apps/worker/src/bootstrap.ts

```diff
@@ -66,15 +66,16 @@ export async function bootstrap(): Promise<INestApplication> {
   app.use(bodyParser.json());
   app.use(bodyParser.urlencoded({ extended: true }));
 
-  // Starts listening for shutdown hooks
   app.enableShutdownHooks();
 
   Logger.log('BOOTSTRAPPED SUCCESSFULLY');
 
-  await app.listen(process.env.PORT);
+  await app.init();
 
   await startAppInfra(app);
 
+  await app.listen(process.env.PORT);
+
```

Comment:
Does anyone know if we need to start listening before we startAppInfra? IMO we first need to set up the app and then start listening to new requests.

I made the same for WS, there i noticed that nest is not initialized properly, and `app.init()` helps to initialize the app without starting to listen for new requests.